gruntfile.js

```diff
@@ -11,6 +11,7 @@ module.exports = function(grunt) {
     babel: {
       options: {
         sourceMap: true,
+        compact: true,
```

Comment:
I'm not 100% sure, but I think we don't want this setting here. I believe the idea is that the babel output is nice and readable, and only the minified build is compact

Reply:
> I'm not 100% sure, but I think we don't want this setting here. I believe the idea is that the babel output is nice and readable, and only the minified build is compact. Let's take it out of this PR - if you think we should change this, let's do that in a separate PR

In that case make it false? Because, by default babel will attempt to make it compact (upto 500KB). And it will throw that warning, as files are above that size. It would be great to have a defined value, rather than uncertain results.

Reply:
Changed compact to false

Reply:
Please review. Thanks